build.gradle

```diff
@@ -103,6 +103,7 @@ ext {
     compileSdkVersion = 34
     minSdkVersion = 21
     minSdkVersionPayPalMessaging = 23
+    versionCode = 186
```

Comment:
We might be able to remove this variable. The Demo app would be the only artifact with a version code. Android started marking the `versionCode` property deprecated for the `.aar` library artifacts we publish to Maven Central.

Reply:
Interesting, this was added back due to accidentally removing it when merging main into the feature branch, but if we can remove this on main eventually we should!